realm/realm-library/src/main/java/io/realm/Realm.java

```diff
@@ -1173,6 +1173,48 @@ public void clear(Class<? extends RealmObject> clazz) {
         getTable(clazz).clear();
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
```

Comment:
You should, but in this case where your override isn't doing anything. You can just leave it out, both editor autocomplete + javadoc will find it correctly.